code/modules/mob/dead/observer/observer.dm

```diff
@@ -109,19 +109,29 @@ var/list/image/ghost_images_simple = list() //this is a list of all ghost images
 	animate(src, pixel_y = 2, time = 10, loop = -1)
 	..()
 
+/mob/dead/observer/narsie_act()
```

Comment:
@KazeEspada: Observers become harvesters when they click on Nar-Sie, and that uses a different proc than `narsie_act()`.